README.md

```diff
@@ -229,7 +229,7 @@ start(task4) // blocks until one of the previous tasks is finished
 
 ## Solo Tasks
```

Comment:
The term annoys me a bit, but getting the word "promise" in here somewhere might help SEO for people looking for "one of those that is correct".

Reply:
Ugh, you're absolutely right, but "promise" and "future" have both been poisoned for me 😆 Probably should at least mention them, though.